pynucastro/networks/rate_collection.py

```diff
@@ -580,6 +583,97 @@ def evaluate_rates(self, rho, T, composition):
 
         return rvals
 
+    def _evaluate_comp_NSE(self, u, rho, T, ye):
+        """ A helper equation that finds the mass fraction of each nuclide in NSE state,
+        u[0] is chemical potential of proton  while u[1] is chemical potential of neutron"""
+
+        # Define constants: amu, boltzmann, planck, and electron charge
+        m_u = constants.value("unified atomic mass unit") * 1.0e3  # atomic unit mass in g
+        k = constants.value("Boltzmann constant") * 1.0e7          # boltzmann in erg/K
+        h = constants.value("Planck constant") * 1.0e7             # in cgs
+        e = 4.8032e-10                                           # electron charge in cgs
+
+        # These are three constants for calculating coulomb corrections of chemical energy, see Calders paper: iopscience 510709, appendix
+        A_1 = -0.9052
+        A_2 = 0.6322
+        A_3 = -0.5 * np.sqrt(3.0) - A_1 / np.sqrt(A_2)
+
+        # Create composition object for NSE and find electron number density
+        comp_NSE = Composition(self.unique_nuclei)
+        n_e = rho * ye / m_u
+
+        # u_c is the coulomb correction term for NSE
+        # Calculate the composition at NSE, equations found in appendix of Calder paper
+        for nuc in self.unique_nuclei:
+            gamma = nuc.Z**(5. / 3.) * e**2 * (4.0 * np.pi * n_e / 3.0)**(1. / 3.) / k / T
+            u_c = 624151.0 * k * T * (A_1 * (np.sqrt(gamma * (A_2 + gamma)) - A_2 * np.log(np.sqrt(gamma / A_2) +
+                                      np.sqrt(1.0 + gamma / A_2))) + 2.0 * A_3 * (np.sqrt(gamma) - np.arctan(np.sqrt(gamma))))
+            comp_NSE.X[nuc] = m_u * nuc.A_nuc * nuc.partition_function(T) / rho * (2.0 * np.pi * m_u * nuc.A_nuc * k * T / h**2)**(3. / 2.) \
+            * np.exp((nuc.Z * u[0] + nuc.N * u[1] - u_c + nuc.nucbind * nuc.A) / k / T * 1.6022e-6)
```

Comment:
instead of hardcoding 1.6022e-6 here, can you create a variable MeV2ergs or something like that?